go/tasks/plugins/hive/execution_state.go

```diff
@@ -27,7 +31,7 @@ const (
 	PhaseNotStarted ExecutionPhase = iota
 	PhaseQueued                    // resource manager token gotten
 	PhaseSubmitted                 // Sent off to Qubole
-
+	PhaseWriteOutputFile
```

Comment:
Why are we adding this? if so we might need to add a transformer in Propeller - https://github.com/lyft/flytepropeller/blob/master/pkg/controller/nodes/task/transformer.go#L41